entries/last-crown/game.json

```diff
@@ -0,0 +1,19 @@
+{
+    "title": "Last Crown Warriors",
+    "slug": "last-crown",
+    "developer": "Light Games",
+    "platform": "GB",
+    "typetag": "game",
+	"description": "",
+	"date": "2019-12-01",
+    "screenshots" : ["screenshot1.png"],
+    "rom" : "lastcrowndemo.gb",
```

Comment:
Maybe it's better to add the `wip` tag as this is a demo?

Reply:
Yup! Added to both demo games.